clang/include/clang/StaticAnalyzer/Core/PathSensitive/CallDescription.h

```diff
@@ -107,7 +107,8 @@ class CallDescription {
     return CD1.matches(Call);
   }
 
-  /// \copydoc clang::ento::CallDescription::matchesAny(const CallEvent &, const CallDescription &)
+  /// \copydoc clang::ento::CallDescription::matchesAny(const CallEvent &, const
+  /// CallDescription &)
```

Comment:
I'm not sure how to resolve this. I'd need to check how to resolve this for Doxygen.
https://clang.llvm.org/doxygen/classclang_1_1ento_1_1CallDescription.html#ac6873d422bacd2cc5a9857b4f96248ff